plugins/webp-uploads/tests/bootstrap.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * PHPUnit bootstrap file.
+ *
+ * @package webp-uploads
+ */
+
+define( 'TESTS_PLUGIN_DIR', dirname( __DIR__ ) );
+define( 'TESTS_PLUGIN_ROOT', dirname( dirname( TESTS_PLUGIN_DIR ) ) );
+
+// Determine correct location for plugins directory to use.
+if ( false !== getenv( 'WP_PLUGIN_DIR' ) ) {
+	define( 'WP_PLUGIN_DIR', getenv( 'WP_PLUGIN_DIR' ) );
+} else {
+	define( 'WP_PLUGIN_DIR', dirname( TESTS_PLUGIN_DIR ) );
+}
+
+// Load Composer dependencies if applicable.
+if ( file_exists( TESTS_PLUGIN_ROOT . '/vendor/autoload.php' ) ) {
+	require_once TESTS_PLUGIN_ROOT . '/vendor/autoload.php';
+}
+
+// Detect where to load the WordPress tests environment from.
+$_tests_dir = WPP_Tests_Helpers::get_path_to_wp_test_dir();
+require_once $_tests_dir . '/includes/functions.php';
+
+// Force plugin to be active.
+$GLOBALS['wp_tests_options'] = array(
+	'active_plugins' => array( basename( TESTS_PLUGIN_DIR ) . '/load.php' ),
+);
+
+// Start up the WP testing environment.
+require $_tests_dir . '/includes/bootstrap.php';
```

Comment:
This code would be shared across all standalone plugins' bootstrap files, correct? I think this would make sense to put up in some common shared tests dir:

```suggestion
require TESTS_PLUGIN_ROOT . '/tests/standalone-bootstrap.php';
```

Reply:
@westonruter I think we should keep the configurable part in the plugin-specific bootstrap file. In the case of some plugin, we need to add some test filter before WP tests bootstrap, then we again need to make changes to this file.

Reply:
We _may_ need to. But if we don't now, maybe err on the side of reducing duplication until we need to?